test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -15,6 +15,7 @@ check_result_qemu() {
     umount $TESTDIR/root
     [[ -f $TESTDIR/failed ]] && cat $TESTDIR/failed
     ls -l $TESTDIR/journal/*/*.journal
+    journalctl -u testsuite --no-pager --file=$TESTDIR/journal/*/*.journal
```

Comment:
@martinpitt 
> once that lands I'll take it out of the Debian test

OK. But I think we should fix the `upstream`-test for now. I prepared a patch: https://gist.githubusercontent.com/evverx/6805f228a00f5cd362d3f0cd9d30f5c6/raw/02c1e080613a3b599174dba5c5029c11f37e35a8/0001-debian-tests-upstream-print-all-journal-files.patch
Please, take a look